### PR TITLE
TSlint updated and errors fixed

### PR DESCRIPTION
**Pull request by kraenhansen**

### CODE REVIEW

File: src/services/data-importer/csv/CSVSchemaGenerator.ts [hunk 1]
```diff
@@ -52,8 +52,10 @@ export class CSVSchemaGenerator extends SchemaGenerator {
         schema.properties[header] = Util.isBoolean(value)
           ? 'bool?'
           : Util.isInt(value)
-            ? 'int?'
-            : Util.isDouble(value) ? 'double?' : 'string?';
+          ? 'int?'
+          : Util.isDouble(value)
```
Comment (nirinchev) [inline src/services/data-importer/csv/CSVSchemaGenerator.ts]: 😱

Comment (kraenhansen) [reply src/services/data-importer/csv/CSVSchemaGenerator.ts]: I didn't write this :D I just ran the linter :P

Comment (nirinchev) [reply src/services/data-importer/csv/CSVSchemaGenerator.ts]: See - even the linter has no idea what to do with the conditional 😂 I'm telling you, we should use plain old if-s 🙄

Comment (kraenhansen) [reply src/services/data-importer/csv/CSVSchemaGenerator.ts]: Agree - for this a plain old if would probably be the best choice :)

**Pull request merged**